lucien-ios/Dashboard/LendingCollectionViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LendingViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/9/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+final class LendingCollectionViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+
+    @IBOutlet weak var collectionView: UICollectionView!
+
+    let reuseIdentifier = "comicCell"
+    let objects = ["Princess", "Gracie", "Kittens"]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.collectionView.register(UINib(nibName:"LendingCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: reuseIdentifier)
+        self.collectionView.delegate = self
+        self.collectionView.dataSource = self
+        setLayout()
+    }
+
+    func setLayout() {
+        let collectionViewLayout = collectionView.collectionViewLayout as? UICollectionViewFlowLayout
+        let sectionInset = UIEdgeInsetsMake(0.0, 24.0, 0.0, 0.0)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Legacy Constructor Violation: Swift constructors are preferred over legacy convenience functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Alegacy_constructor)

Reply:
Do you know what this error message means?

Reply:
I think it means I used the old obj C way of setting insets -- xcode automatically deletes the parameter descriptions top, left, right, bottom so I input the numbers as is; I'll fix this!